src/ui/Histogram/Histogram.js

```diff
@@ -0,0 +1,692 @@
+/*
+ * Copyright 2022 EPAM Systems, Inc. (https://www.epam.com/)
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useRef, useState, useEffect } from 'react';
+import TransfFunc from '../../engine/TransFunc';
+
+const DEFAULT_HEIGHT = 220;
+const NEED_TO_DRAW_VERTICAL_MARKS = false;
+
+const Histogram = (props) => {
+  const canvasRef = useRef(null);
+  const canvasHistogram = useRef(null);
+  const [width, setWidth] = useState(0);
+  const [height, setHeight] = useState(DEFAULT_HEIGHT);
+  let m_numColors = 0;
```

Comment:
I think `m_` has no sense :) Please replace `m_numColors` to basic `numColors`, and all occurrences `m_`.